benchmark/test_special_perf.py

```diff
@@ -140,6 +140,37 @@ def padding_kwargs(dtype, batch, size):
     bench.run()
 
 
+def test_perf_upsample_bicubic2d_aa():
+    def upsample_bicubic2d_aa_kwargs(dtype, batch, size):
+        channel = 16
+        scale_factors = (2, 2)
+        shape = (batch, channel, size, size)
+        output_size = (
+            int(shape[2] * scale_factors[0]),
+            int(shape[3] * scale_factors[1]),
+        )
+        input = torch.randn(size=shape, device="cuda", dtype=dtype)
+        return {
+            "input": input,
+            "output_size": output_size,
+            "align_corners": False,
+            "scales_h": None,
+            "scales_w": None,
+        }
+
+    bench = Benchmark(
+        op_name="_upsample_bicubic2d_aa",
+        # py api torch.nn.functional.interpolate,
+        torch_op=torch._C._nn._upsample_bicubic2d_aa,
+        arg_func=None,
+        dtypes=[torch.float16, torch.float32],
```

Comment:
Is there any problem with bfloat16

Reply:
the bf16 type has a minor accuracy issue，like this
```
AssertionError: Tensor-likes are not close!
    Mismatched elements: 1 / 1871310 (0.0%)    
    Greatest absolute difference: 0.01171875 at index (1, 1, 265, 43) (up to 0.0025 allowed)    
    Greatest relative difference: 0.02099609375 at index (1, 1, 265, 43) (up to 0.016 allowed)  
````